wasm/build.js

```diff
@@ -260,6 +184,11 @@ async function build(network) {
         buildJS(network),
         buildWorker(network),
     ]);
+
+    await Promise.all([
+        $fs.rm($path.join("dist", network, "tmp"), { recursive: true }),
+        $fs.rm($path.join("dist", network, "aleo_wasm_custom.d.ts")),
+    ]);
```

Comment:
What does this do ultimately?

Reply:
The build process generates some files which aren't needed, so it cleans them up in order to reduce the size of the npm package.